src/Compilers/VisualBasic/Portable/project.json

```diff
@@ -1,6 +1,8 @@
 {
-  "dependencies": { },
+  "dependencies": {
+    "System.Threading.Tasks.Parallel": "4.0.1"
```

Comment:
Need a reference to `NETStandard.Library` and `Microsoft.NETCore.Portable.Compatibility`